src/NUnitFramework/framework/Interfaces/TNode.cs

```diff
@@ -22,54 +22,44 @@
 // ***********************************************************************
 
 using System;
+using System.Xml;
+
+#if PORTABLE || SILVERLIGHT
+using System.Xml.Linq;
+#endif
 
 namespace NUnit.Framework.Interfaces
 {
     /// <summary>
-    /// XmlNode represents a single node in the XML representation
+    /// TNode represents a single node in the XML representation
     /// of a Test or TestResult. It replaces System.Xml.XmlNode and
-    /// provides a minimal set of methods for operating on the XML 
-    /// in a platform-independent manner.
+    /// System.Xml.Linq.XElement, providing a minimal set of methods 
+    /// for operating on the XML in a platform-independent manner.
     /// </summary>
-    public class XmlNode
+    public class TNode
```

Comment:
The whole XML namespace is way too polluted with different classes for working with XML, so it doesn't leave much does it. NNode (Nunit) doesn't read well either. The only thing that I can think of is going longer like NunitXmlNode, but that doesn't really flow off the tongue either does it. We should probably leave it as is unless one of us suddenly has an epiphany :smile: